protosocket-prost/src/prost_client_registry.rs

```diff
@@ -9,6 +9,7 @@ use crate::{ProstClientConnectionBindings, ProstSerializer};
 #[derive(Debug, Clone)]
 pub struct ClientRegistry<TConnector = TcpConnector> {
     max_buffer_length: usize,
+    buffer_allocation_interval: usize,
```

Comment:
You used "increment" in the Connection and SocketServer. I think that's the better word, and it should be consistent here and elsewhere.

Reply:
no worries, I was pretty sure it was just a mistype anyway